src/packages/ListenLiveClient.ts

```diff
@@ -34,6 +35,13 @@ export class ListenLiveClient extends AbstractLiveClient {
   ) {
     super(options);
 
+    if (
```

Comment:
I was debating if we needed this exception in the SDKs as our API enforcing it.

Reply:
Discussed in Slack - moot point, we're fine either way as these SDKs will soon be approached differently.